base_export_manager/views/ir_model_view.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0"?>
+<odoo>
+
+    <record id="view_model_form_base_export_manager" model="ir.ui.view">
```

Comment:
Please follow the [Naming xml_id guidelines](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#naming-xml_id) for these Ids